exchanges/bittrex/bittrex_wrapper.go

```diff
@@ -954,7 +954,7 @@ func (b *Bittrex) FormatExchangeKlineInterval(in kline.Interval) string {
 // This implementation rounds returns candles up to the next interval or to the end
 // time (whichever comes first)
 func (b *Bittrex) GetHistoricCandles(ctx context.Context, pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time) (*kline.Item, error) {
-	req, err := b.GetKlineRequest(pair, a, interval, start, end)
+	req, err := b.GetKlineRequest(pair, a, interval, start, end, false)
```

Comment:
This one is technically true:
> Retrieve recent candles for a specific market and candle interval. The maximum age of the returned candles depends on the interval as follows: (MINUTE_1: 1 day, MINUTE_5: 1 day, HOUR_1: 31 days, DAY_1: 366 days)

https://bittrex.github.io/api/v3 but I understand that it makes your simpler single `bool` approach more difficult